db/models.js

```diff
@@ -60,16 +57,6 @@ exports.fetchCommentsById = (queryId) => {
     })
 }
 
-
-
-// const comments = result.rows[0];
-// if (!comments) {
-// 	return Promise.reject({
-// 	  status: 404,
-// 	  msg: `The id number ${queryId}, is not currently in use`,
-// 	});
-//   }
-// return user
-
-
-//basically the test we used in 'GET/api/reviews/:review_id' was to check that the rows were 0 and in that case return an error, the problem is that now rows containing 0 is a possiblity if that person has not made any comments. - Therefore I need to find a way to check that that review_id exists as a condition 
\ No newline at end of file
+exports.postCommentsById = (postRequest, queryId) => {
+    console.log(postRequest, queryId)
```

Comment:
This shouldn't be here either but I think you did go back and delete it afterwards :)